src/content/loaders/index.md

```diff
@@ -26,14 +27,15 @@ Loaders are activated by using `loadername!` prefixes in `require()` statements,
 * `cson-loader` Loads and transpiles a [CSON](https://github.com/bevry/cson#what-is-cson) file
 
 
-## Transpiling
+## Dialects
```

Comment:
I'd consider "[transpiling](https://en.wikipedia.org/wiki/Source-to-source_compiler)" more concise than "dialect" and stick with this term.